js/menu.toggle.js

```diff
@@ -1,68 +1,92 @@
-$(function(){
-	$('#menu--toggle').click(function(){
-		$(this).toggleClass('active');
-		$(this).next().toggleClass('active');
-		console.log('clicked');
-		if ($(this).hasClass('active')) {
-			$(this).text('Hide Menu');
-		}
-		else {
-			$(this).text('View Menu');
-		}
-	});
-});
+// Detect keyboard navigation to setup pseudo classes for focus to allow for
+// only setting focus for keyboard nav
+// https://medium.com/hackernoon/removing-that-ugly-focus-ring-and-keeping-it-too-6c8727fefcd2
+// This works in conjunction with adding a focus element for .main-nav-link
+// if this `user-is-tabbing` exists on body.
+function handleFirstTab(e) {
+    if (e.keyCode === 9) { // the "I am a keyboard user" key
+        document.body.classList.add('user-is-tabbing');
+        window.removeEventListener('keydown', handleFirstTab);
+    }
+  } 
+window.addEventListener('keydown', handleFirstTab);
 
-// why are there two of these toggles?
+// Toggle hamburger menu on mobile displays.
 $('header .menu--toggle').click(function(){
-	$('#nav-main').toggleClass('active');
-	$('.wrap-page').toggleClass('mobile-nav-active');
-});
+    // This toggles the aria-hidden value on the mobile menu.
+    $("#nav-mobile").attr("aria-hidden", function (i, attr) {
+        return (
+            "true" === attr ? "false" : "true"
+        );
+    })
+    // These toggle the CSS classes controlling mobile menu visibility.
+    $("#nav-mobile").toggleClass('active');
+    $('.wrap-page').toggleClass('mobile-nav-active');
 
+    // toggle link visibility (important to keep them out of tabindex until needed)
+    $(".mobile-nav-link").toggleClass('hide-mobile-nav-link');
+    $(".mobile-nav-link").toggleClass('main-nav-link');
+});
 
-$( '.link-primary' ).bind( "mouseenter", function() {	
-	$( '.link-primary' ).removeClass( 'open' );
-	$(this).find( '.menu-control' ).attr( 'aria-expanded', 'true' );
-	$(this).closest( '.link-primary' ).addClass( 'open' );
+// Show or hide the flyout menus on main navigation in response to
+// mouseenter / mouseleave
+$( '.link-primary' ).bind( "mouseenter", function() {
+    $( '.link-primary' ).removeClass( 'open' );
+    $(this).find( '.menu-control' ).attr( 'aria-expanded', 'true' );
+    $(this).closest( '.link-primary' ).addClass( 'open' );
 });
-$( '.link-primary' ).bind( "mouseleave", function() {	
-	$(this).find( '.menu-control' ).attr( 'aria-expanded', 'false' );
-	$( '.link-primary' ).removeClass( 'open' );
+$( '.link-primary' ).bind( "mouseleave", function() {
+    $(this).find( '.menu-control' ).attr( 'aria-expanded', 'false' );
+    $( '.link-primary' ).removeClass( 'open' );
 });
 
-// make esc close all menus
-$( '#nav-main' ).on( 'keydown' , function(e) {
-	if (e.keyCode == 27) {
-		hideMenu(e);
-	}
-});
+// Make ESC close all menus.
+// TODO: when the user has NVDA open, this will not work - but menus can
```

Comment:
This appears to no longer be true - I just tested with NVDA open, and the conflict I was remembering doesn't exist. I can push a commit to remove this note.